pysmartdatamodels/test/test_pysmartdatamodels.py

```diff
@@ -12,40 +12,74 @@
 
```

Comment:
I was expecting that we could include some asserts to make sure that it works as expected and that we can run it as well on a regular basis